examples-gallery/plot_parameter_estimation.py

```diff
@@ -0,0 +1,209 @@
+# %%
+"""
+
+Parameter Identification of a Mass-Spring-Damper System.
+========================================================
```

Comment:
I'd prefer "Parameter Identification from Non-Continuous Measurements" of something similar. The unique aspect of this example relative to what we already have is that we have these independent measurements of the system's motion.

Reply:
Maybe "non-contiguous" is the right word.

Reply:
Parameter Identification from Non-Contiguous Measurements

Reply:
> I'd prefer "Parameter Identification from Non-Continuous Measurements" of something similar. The unique aspect of this example relative to what we already have is that we have these independent measurements of the system's motion.

Will do so and push

Reply:
Also change the filename to `plot_non_contiguous_parameter_estimation.py`